src/main/java/seedu/duke/salesmanager/SalesManager.java

```diff
@@ -0,0 +1,66 @@
+package seedu.duke.salesmanager;
+
+import seedu.duke.model.ContainerList;
+import seedu.duke.model.Item;
+import seedu.duke.model.ItemContainer;
+import seedu.duke.model.exception.DuplicateItemException;
+import seedu.duke.model.exception.IllegalArgumentException;
+import seedu.duke.model.exception.ItemContainerNotExistException;
+import seedu.duke.model.exception.ItemNotExistException;
+
+import java.time.LocalDateTime;
+
+public class SalesManager {
+
+    private static SalesManager salesManager;
+    private ItemContainer soldItems;
+
+    private SalesManager() {
+        ContainerList containerList = ContainerList.getContainerList();
+        try {
+            soldItems = containerList.getContainer("soldItems");
+        } catch (ItemContainerNotExistException e) {
+            // this is why i want to make the model exceptions RuntimeExceptions
+            try {
+                soldItems = containerList.addContainer("soldItems");
```

Comment:
I think i get why you want to catch RunTimeException in this case, since the sold itemContainer name is always fixed and that an exception would only occur if there is any other exception, other that illegal argument exception  in creating the itemcontainer, but then i am not sure if it would comply with the coding practices.

Reply:
Ok so i read the textbook again and I feel that this is only a special case where the exception should not occur, so it should still be Exception, not RuntimeException.

Reply:
Yep, i think Exception would be better